selinux/ipa.te

```diff
@@ -412,3 +420,28 @@ optional_policy(`
 optional_policy(`
        systemd_private_tmp(ipa_custodia_tmp_t)
 ')
+
+optional_policy(`
+    gen_require(`
+        type tomcat_t;
```

Comment:
To avoid using types from other policy modules, it's desired to use interfaces from the modules. It is acceptable as a temporary solution, but we should avoid it long-term. In this case, an interface needs to be defined in ipa.if and called from tomcat.te.

Reply:
Note the same goes for #L344-L346.

Reply:
OK, we will remove it as time permits. For now I suspect we'll have to keep it.

Reply:
@zpytela So we keep these as a temporary solution for now.